contrib/node/src/python/pants/contrib/node/targets/node_module.py

```diff
@@ -53,7 +59,8 @@ def __init__(
       'package_manager': PrimitiveField(package_manager),
       'output_dir': PrimitiveField(output_dir),
       'dev_dependency': PrimitiveField(dev_dependency),
-      'style_ignore_path': PrimitiveField(style_ignore_path)
+      'style_ignore_path': PrimitiveField(style_ignore_path),
+      'bin_executables': PrimitiveField(bin_executables)
```

Comment:
Recommend adding trailing commas to avoid (in future) the extra diff you see here on the previous line.